inspire/dojson/jobs/schemas/jobs-0.0.1.json

```diff
@@ -33,15 +33,15 @@
             "type": "array",
             "uniqueItems": true
         },
-        "deadline": {
+        "deadline_date": {
             "format": "date",
             "title": "Application deadline",
             "type": "string"
         },
         "experiment": {
             "description": "FIXME: we can simply store here the ID of the experiment.",
-            "title": "Experiment recid",
-            "type": "integer"
+            "title": "Experiment associated with Job",
+            "type": "string"
```

Comment:
We should rather use a $ref to experiment :-) maybe this could be done in one PR for all schemas.